source/read-document-playground.txt

```diff
@@ -0,0 +1,154 @@
+.. _vsce-read-doc-playground:
+
+==============
+Read Documents
+==============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can read documents in a collection using the 
+:manual:`MongoDB CRUD Operators </crud>` in a MongoDB Playground:
+
+- Use the :manual:`findOne() 
+  </reference/method/db.collection.findOne>` method to read one
+  document.
+- Use the :manual:`find() 
+  </reference/method/db.collection.find>` method to read more 
+  than one document.
+
+Prerequisites
+-------------
+
+If you have not done so already, you must complete the following 
+prerequisites before you can read documents with a MongoDB Playground:
+
+- :ref:`Create a connection to a MongoDB deployment 
+  <vsce-connect-task>`.
+- :ref:`Activate the connection to the MongoDB deployment 
+  <vsce-activate-connection>`.
+- :ref:`Open a MongoDB Playground <open-playground-for-crud-vsce>`.
+- :ref:`vsce-create-doc-playground` or create documents in a collection
+  using a different method.
+
+Read One Document
+-----------------
+
+To read one document, use the following syntax in your Playground:
+
+.. code-block:: javascript
+
+   db.collection.findOne(
+      { <query> },
+      { <projection> }
+   )
+
```

Comment:
I'd suggest including the behavior that occurs when multiple documents are matched.

```
... returns the first document according to the natural order which reflects the order of documents on the disk.
```

https://docs.mongodb.com/manual/reference/method/db.collection.findOne/#db.collection.findOne